func.go

```diff
@@ -411,6 +420,7 @@ func (f *nativeFuncObject) assertCallable() (func(FunctionCall) Value, bool) {
 	return nil, false
 }
 
+// todo Might can remove this as a duplicate
```

Comment:
I think vmCall (below) might remove the need for this function

Reply:
I really like the `goja-improvements` label idea. I went ahead and associated our "not-doing" tickets with that label.

Reply:
Perfect, sounds like a good plan to me!